src/Magnum/GL/Test/TransformFeedbackGLTest.cpp

```diff
@@ -488,9 +489,10 @@ void TransformFeedbackGLTest::attachRanges() {
         .setCount(2);
 
     TransformFeedback feedback;
+    using BufferOffset = Containers::Triple<Buffer*, GLintptr, GLsizeiptr>;
     feedback.attachBuffers(0, {
-        std::make_tuple(&output1, 256, 2*sizeof(Vector2)),
-        std::make_tuple(&output2, 512, 2*sizeof(Float))
+        BufferOffset{&output1, 256, 2*sizeof(Vector2)},
+        BufferOffset{&output2, 512, 2*sizeof(Float)}
```

Comment:
That makes the MAGNUM_BUILD_DEPRECATED build of this test fail, since the call is ambiguous (could be either the std::initializer_list variant or the ArrayView variant)

Reply:
Hmm whoops, no actually you were right, but only once I removed the `<tuple>` include, the expression is not ambiguous anymore! So, done